src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar-wrapper">
+      <main class="calendar calendar--month-length--29">
+        <div class="calendar__day calendar__day--1 calendar__day--start-day-sunday"></div>
+        <div class="calendar__day calendar__day--2"></div>
```

Comment:
You don't need calendar__day--$ modifier. Reference each day through css `:nth-child property`